content/docs/start/index.md

```diff
@@ -95,31 +108,24 @@ The steps and results of some of these chapters are captured in our
 
 </admon>
 
-## Experiments
-
-- **[Experiment versioning]**
-
-  Track the changes to the code, data, metrics, parameters and plots associated
-  with each experiment, without bloating your Git repo.
+## Experiment Management
 
-- **[Experiment management]**
+- **[Experiment live tracking]** - Instrument your code to quickly start
+  tracking experiments. Manage changes to code, data, metrics, parameters and
+  plots associated with each experiment without bloating your Git repo.
 
-  Manage experiments and share them with others using software engineering best
-  practices.
+- **[Experiment pipelines]** - Leverage DVC data pipelines as an experiment
+  management system. Split your workflow into stages, track dependencies and
+  outputs, and natively inject parameters and hyper-parameters to experiment
+  runs.
 
-- **[Building pipelines]**
+- **[Experiment collaboration]** - Manage experiments the same way developers
+  manage code. Share experiments with your team over git, create branches and
+  pull requests, and compare results.
```

Comment:
I think @daavoo had put some of the exp mgmt stuff before pipelines for a couple reasons:
1. To make clear that pipelines weren't a prerequisite
2. Because exp mgmt/collaboration might be a more important value prop than the exp pipelines

If you are trying to keep these pages somewhat independent, WDYT about switching the order here?

Reply:
I think it might be a good time to make `Pipelines` a separate component

Reply:
@dberenbaum  - reordered

Reply:
As a counterpoint, DVC pipelines was the primary value prop for me adopting it as a user. It's not a replacement for Airflow, but that's not what I wanted during development where everything changed frequently. As you mentioned above @omesser, it's more like a build tool for ML. Framed that way, I think is has a lot of value.

Reply:
> As a counterpoint, DVC pipelines was the primary value prop for me adopting it as a user.

That's a good point. I'm still on the fence - we "want" to sell this, but I feel pretty strongly about people having different connotation when they see "pipelines". In the context of exp management (usually single stage stupid task managers for competitors) - it's a powerful implementation. In the context of pipeline orchestrators - we don't compete (or, don't compete that well, imo).

Reply:
@daavoo  - nothing to apologize for. a great point, I hope it will drive the next PR here and then we'll attack this problem closer to its "core"

Reply:
For now, I re-ordered as I said, so resolving this thread